README.md

```diff
@@ -168,6 +168,31 @@ rvm use ruby-2.3.4@rails5.0.5 --create
  [rbenv-gemset](https://github.com/jf/rbenv-gemset)
 </details>
 
+<details>
+  <summary>3) Install Yarn (click to expand)</summary>
+
+# Install Yarn
+
+There are step-by-steps [here](https://yarnpkg.com/en/docs/install) for all of the major operating systems.
+
+```
+cd client/
+yarn install
+yarn flow
+```
+
+## NPM Packages
+Some NPM packages have flow type enabled but fail the flow checks (e.g. radium). You'll want to put the package path under the `[ignore]` section of `.flowconfig`, for example:
```

Comment:
Technically, the reason they fail is because those NPM packages use an older version of flow, which is why we want to ignore them. It will be some time before we can figure out how to actually support different flow versions within package dependencies. I think just add for clarification "because it uses an older flow version compared to the main project".